ext/ffi/ffi.c

```diff
@@ -974,7 +974,7 @@ static void zend_ffi_callback_trampoline(ffi_cif* cif, void* ret, void** args, v
 	}
 
 	if (callback_data->arg_count) {
-		int n = 0;
+		uint32_t n = 0;
```

Comment:
its scope seems limited to the loop, you can probably declare it in the for loop like elsewhere.

Reply:
Ah yes indeed.